test/cljs/status_im/test/contacts/handlers.cljs

```diff
@@ -1,13 +1,11 @@
 (ns status-im.test.contacts.handlers
+  (:require-macros [day8.re-frame.test :refer [run-test-sync]])
   (:require [cljs.test :refer-macros [deftest is testing]]
             reagent.core
             [re-frame.core :as rf]
-            [day8.re-frame.test :refer-macros [run-test-sync]]
-            status-im.specs
             status-im.db
             [status-im.contacts.events :as contacts-events]
             [status-im.group.events :as group-events]
-            [status-im.handlers :as handlers]
```

Comment:
no problem, got bitten hard by these namespaces cleanup in my own PR because clj-refactor removed the `status-im.subs` import that was unused in the core ns but necessary for the app to work :D